pulp_rpm/app/models.py

```diff
@@ -528,8 +580,7 @@ class UpdateCollection(Model):
     shortname = models.TextField()
     module = models.TextField(default='')
 
-    update_record = models.ForeignKey(UpdateRecord, related_name="collections",
-                                      on_delete=models.CASCADE)
+    update_record = models.ManyToManyField(UpdateRecord, related_name="collections")
```

Comment:
This is needed not to duplicate collections when we merge advisories.

Migrations are squashed due to this change.